sdk/communication/azure-communication-chat/samples/chat_thread_client_sample_async.py

```diff
@@ -80,24 +80,46 @@ async def create_chat_thread_client_async(self):
         print("thread created, id: " + self._thread_id)
         print("create_chat_thread_client_async succeeded")
 
+    async def get_chat_thread_properties_async(self):
+        thread_id = self._thread_id
+        # [START get_thread]
+        from azure.communication.chat.aio import ChatClient
+        from azure.communication.identity._shared.user_credential_async import CommunicationTokenCredential
+        from azure.communication.chat._shared.user_token_refresh_options import CommunicationTokenRefreshOptions
```

Comment:
The credentials objects should just be imported from `azure.communication.chat` now right?

Reply:
yes. The changes were in a dangling in a separate PR #17178 . It's been merged.